sbin/woeusb

```diff
@@ -2076,18 +2076,18 @@ is_var_set_and_not_empty(){
 # Utility function to check if function parameters quantity is legal
 # NOTE: non-static function parameter quantity(e.g. either 2 or 3) is not supported
 check_function_parameters_quantity(){
-    if [ "${#}" -ne 3 ]; then
+    if [ "${#}" -ne 2 ]; then
         print_fatal \
-            'Function %s (line %u) requires %u parameters, but %u is given\n' \
-            ${FUNCNAME[0]} \
-            ${LINENO} \
-            3 \
+            'Function %s (%s) requires %u parameters, but %u is given\n' \
```

Comment:
I am enlightened by the usage of the `caller` built-in command, however, I don't feel that the format of its output is desirable so I'll pass on this one.